version/version.go

```diff
@@ -15,3 +21,63 @@ func PrintVersion() {
 	version := CurrVersion
 	fmt.Println(cliCurrentVersion + version)
 }
+
+func getLatestRelease(client *github.Client, owner, repo string) (*github.RepositoryRelease, error) {
+	ctx := context.Background()
+
+	// Get the list of releases for the repository
+	releases, _, err := client.Repositories.ListReleases(ctx, owner, repo, nil)
+	if err != nil {
+		return nil, err
+	}
+
+	// Find the latest release (by release tag)
+	var latestRelease *github.RepositoryRelease
+	for _, release := range releases {
+		// Skip pre-releases
+		if release.GetPrerelease() {
+			continue
+		}
+		if latestRelease == nil || release.GetTagName() > latestRelease.GetTagName() {
+			latestRelease = release
+		}
+	}
+
+	return latestRelease, nil
+}
+
+func CompareVersions(client *github.Client, owner, repo string) error {
```

Comment:
ideally, we shouldn't be returning errors from here, because in case there is some issue with the check we should still be executing the actual command, rather than failing here and blocking everything for the user.

Reply:
can we log the error?

Reply:
fixed